client/dev-tools/components/dev-tools.tsx

```diff
@@ -77,6 +77,34 @@ const DevTools = () => {
 			),
 			supportContext: 'github-deployments',
 		},
+		{
+			title: translate( 'Theme File Editor' ),
+			text: translate(
+				'Unlock access to the theme file editor, a powerful way to edit the individual CSS and PHP files of your theme.'
+			),
+			supportContext: 'theme-file-editor',
+		},
+		{
+			title: translate( 'Plugin File Editor' ),
+			text: translate(
+				'Unlock the plugin file editor and make changes to any of your plugins’ individual PHP files.'
+			),
+			supportContext: 'use-your-pluginsuse-the-plugin-file-editor-advanced',
```

Comment:
![badge](https://img.shields.io/badge/review-ask-blue.svg)

The "Learn more" link is not shown on my side.
<img width="861" alt="Screenshot 2024-05-29 at 16 56 53" src="https://github.com/Automattic/wp-calypso/assets/5287479/81fa8549-68bd-4c93-bc86-a8fc36da94cf">

I did not look into it, but maybe we need to change this key to show the link?

```suggestion
			supportContext: 'use-the-plugin-file-editor-advanced',
```